.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Shouldn't be necessary with a [gobal .gitignore](http://stackoverflow.com/questions/7335420/global-git-ignore)